src/Compilers/Core/MSBuildTask/Csc.cs

```diff
@@ -199,6 +200,13 @@ protected override string ToolNameWithoutExtension
         /// </summary>
         protected override void AddResponseFileCommands(CommandLineBuilderExtension commandLine)
         {
+            // Pass sdkpath if we are invoking core compiler from framework to preserve the behavior that framework compiler would have.
+            // Pass this option only to the built-in compiler (customer-supplied compiler might not support it).
```

Comment:
I am somewhat confused by this, if user-supplied compiler doesn't respect this option, don't we have a reason to believe their compiler will not behave correctly? So why avoid passing the argument?

Reply:
I guess you are right. I was worried about this causing another regression (the custom compiler failing with unknown argument).